src/io.pl

```diff
@@ -0,0 +1,52 @@
+:-consult('validators.pl').
+:-consult('utils.pl').
+
+ask_for_piece_to_add(Player):-
+        repeat,
```

Comment:
syntatic_sugar doesn't seems good. The better option is to add a new predicate that calls again the ask_for_piece_to_add(_), which will make the function to repeat until it succeeds.